### ISSUE TITLE
When i used the sd_setImageWithURL options:SDWebImageRefreshCached,it would not load  the lasted image. may be that i have used the same imageUrl even if the image on server has changed.

### ISSUE THREAD

**Issue by ws752958369** [rs/SDWebImage#1787]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 10.0
 SDWebImage Version      | e.g. 3.8.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 8.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by ws752958369** [comment]
i hope that this problem could be solved as soon.

**Comment by bpoplauschi** [comment]
@ws752958369 take a look at similar issues like #1029. You can do a search using `SDWebImageRefreshCached` as query and find all the related discussions to this topic. 
This is why the issue template contains (to prevent duplicating issues):
> I have searched for a similar issue in the project and found none

Also, please make sure you debug your server properly and that it responds correctly to all queries.

**Comment by bpoplauschi** [comment]
@ws752958369 also, you should fill in all the info in the template because you left it as default.
It's important for us to know if you can reproduce the issue all the time, with our demo or only with a demo project of your own, if you can share some urls that trigger this situation, ...

**Comment by ws752958369** [comment]
Thanks,use  #951 (comment) solutions,it does work well.

**Comment by bpoplauschi** [comment]
Glad to hear that @ws752958369. Closing, based on previous comment.
